arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -62,6 +60,7 @@ public class RomLex
       new ModuleOperationStackedSet<>();
 
   @Getter private List<RomOperation> sortedOperations;
+  Map<ContractMetadata, List<RomOperation>> lookup = new HashMap<>();
```

Comment:
The name lookup is not the right one. It's just a dictionary between CFI <=> RomOperation <=> contract Metadata

Reply:
this map is just used within this class to speed up the loop on sortedOperations. There is no business logic related to it. 

Instead of iterating through it for all getCodeFragmentIndexByMetadata, we precompute it once and reuse it. 

And actually we can even make it better by keeping just the size of the list.